SIS_import/sis_set_url.rb

```diff
@@ -196,7 +183,7 @@ def get_all_canvas_page_urls(response_headers)
 		end
 	}
 
-	p last_page_number
+	@logger.info last_page_number
```

Comment:
This should give some idea of what value is being logged.  All the logger statements should be reviewed to check that they are clearly identifying the values printed.